geoapi/utils/agave.py

```diff
@@ -51,27 +56,36 @@ def uuid(self):
 class AgaveUtils:
     BASE_URL = 'http://api.prod.tacc.cloud'
 
-    def __init__(self, jwt):
-        self.jwt = jwt
+    def __init__(self, jwt=None, token=None, tenant=None):
         client = requests.Session()
-        client.headers.update({'X-JWT-Assertion-designsafe': jwt})
+        if jwt:
+            client.headers.update({'X-JWT-Assertion-designsafe': jwt})
+        if token:
+            client.headers.update({'Authorization': 'Bearer {}'.format(token)})
+        self.base_url = get_api_server(tenant) if tenant else self.BASE_URL
```

Comment:
@reptillicus  Realizing that we talked about always using api.prod.  should i make it optional if we use the tenant's api server (it is needed for using service accounts to get system roles)?